docs/Makefile

```diff
@@ -1,11 +1,10 @@
 docs-install:
 	gem install jekyll bundler
 	bundle install --gemfile=Gemfile
-
+# To run localy you need to use ruby version =< 2.7.6, but not >=3.x , see https://bbs.archlinux.org/viewtopic.php?pid=1976408#p1976408
+#
 # run local server for documentation website
 # at http://127.0.0.1:4000/
 # On first use, please run `make docs-install`
 docs-up:
-	JEKYLL_GITHUB_TOKEN=blank PAGES_API_URL=http://0.0.0.0 bundle exec \
-		--gemfile=Gemfile \
-	 	jekyll server --livereload
+  JEKYLL_GITHUB_TOKEN=blank PAGES_API_URL=http://0.0.0.0 bundle exec --gemfile=Gemfile jekyll serve --livereload
```

Comment:
> it was jekyll server and now it is jekyll serve - are u sure it is correct?
Because [Jekyll docs](https://jekyllrb.com/docs/#instructions) use `jekyll serve`.
> also, why did u change the multiline to single line?
Ops =) 
`make docs-up` didn't work for unknown reasons and that's why I've used single line... I'll revert this changes